Makefile

```diff
@@ -22,3 +22,11 @@ run-ci-e2e-test:
 .PHONY: clean
 clean:
 	rm -rf ${OUTPUT_DIR}
+
+.PHONY: run-local
```

Comment:
Can also be `local-run` and `local-clean`.

Reply:
I vote for `local-*`